src/Sulu/Component/Security/Authorization/AccessControl/AccessControlManager.php

```diff
@@ -62,8 +62,8 @@ public function __construct(
         MaskConverterInterface $maskConverter,
         EventDispatcherInterface $eventDispatcher,
         SystemStoreInterface $systemStore,
-        iterable $descendantProviders = [],
-        RoleRepositoryInterface $roleRepository
+        RoleRepositoryInterface $roleRepository,
+        iterable $descendantProviders = []
```

Comment:
we should not swap any parameter in the constructor. In this case I would just remove the `= []` as it doe snot make any sense